.golangci.yml

```diff
@@ -22,6 +22,11 @@ linters-settings:
       - octalLiteral
       - whyNoLint
       - wrapperFunc
+    settings:
+      rangeValCopy:
+        sizeThreshold: 512
+      hugeParam:
+        sizeThreshold: 512
```

Comment:
with 64 bytes there would be 28 issues,
128 bytes = 17 issues,
256 bytes = 15 issues

However, some of the issues are in test files, maybe we could skip them?

Reply:
But when I think about this, using pointer instead of value should be better most of the time and it does not make the code more complicated, so I would lean towards 64 bytes in this case anyway. What do you think?